docs/built-ins.md

```
@@ -306,10 +306,53 @@ String constructor available as `String()`.
+| `exec(string)` | Returns a match array with capture groups, `index`, and `input`, or `null` |
+| `test(string)` | Returns `true` if the regex matches |
+| `toString()` | Returns `/pattern/flags` |
+| `[Symbol.match](string)` | Matching hook used by `String.prototype.match()` |
+| `[Symbol.matchAll](string)` | Match-all hook used by `String.prototype.matchAll()` |
+| `[Symbol.replace](string, replaceValue)` | Replacement hook used by `String.prototype.replace()` and `replaceAll()` |
+| `[Symbol.search](string)` | Search hook used by `String.prototype.search()` |
+| `[Symbol.split](string, limit?)` | Split hook used by `String.prototype.split()` |
+
+**Behavior notes:**
+
+- Replacement strings in regex-backed `replace()` and `replaceAll()` support `$$`, `$&`, ``$``` , `$'`, and numeric captures such as `$1`.
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)